.eslintrc.json

```diff
@@ -1,3 +1,29 @@
 {
-  "extends": "next/core-web-vitals"
+    "env": {
+        "browser": true,
+        "es2021": true
+    },
+    "extends": [
+        "airbnb",
+        "airbnb-typescript",
+        "next",
+        "prettier"
```

Comment:
Be careful with mixing prettier in like this! I believe it turns off eslint rules that conflict with prettier, but I've always found that running prettier and eslint on the same file can be a pain!

Reply:
I followed a guide about this and they said it'd be okay to mix the 2, but I'll bear that in mind for the future.

For now, should I remove Prettier?

Reply:
Yep, remove it for now!

Reply:
Done this :)